tests/psimulate/test_appending_perf_logs.py

```diff
@@ -0,0 +1,205 @@
+import os
+from datetime import datetime
+from math import ceil
+from pathlib import Path
+
+import pandas as pd
+import pytest
+from _pytest.logging import LogCaptureFixture
+from loguru import logger
+
+from vivarium_cluster_tools.psimulate.paths import OutputPaths
+from vivarium_cluster_tools.psimulate.performance_logger import (
+    append_child_job_data,
+    append_perf_data_to_central_logs,
+    generate_runner_job_data,
+    transform_perf_df_for_appending,
+)
+
+
+@pytest.fixture
+def artifact_perf_df():
+    filepath = Path(__file__).parent / "data/artifact_perf_df.csv"
+    df = pd.read_csv(filepath)
+    index_cols = ["host", "job_number", "task_number", "draw", "seed"]
+    scenario_cols = [col for col in df.columns if col.startswith("scenario")]
+    df = df.set_index(index_cols + scenario_cols)
+    return df
+
+
+@pytest.fixture
+def artifactless_perf_df():

```

Comment:
you might be able to simplify a little bit with something like
```
@pytest.fixture
def df_name():
    return "artifact_perf_df"

@pytest.fixture
def perf_df(df_name):
    filepath = Path(__file__).parent / f"data/{df_name}.csv"
    df = pd.read_csv(filepath)
    index_cols = ["host", "job_number", "task_number", "draw", "seed"]
    scenario_cols = [col for col in df.columns if col.startswith("scenario")]
    df = df.set_index(index_cols + scenario_cols)
    return df
```
and then maybe not need to do request.getfixturevalue(df_name) when you parameterize later?